File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/Patterns/pattern1.cpp
#include <iostream>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    
   //write your code here
   for(i=0;i<n;i++)
   {
       for (j  = 0; j <= i; j++)
       {
           cout << "*\t";
       }
       cout << "\n" ;
   }
   return 0;
}

File: Android/divyaagarwal24_DivyaAgarwal_2024it1100_2/Patterns/pattern1.java
import java.util.*;

public class pattern1 {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int n,i,j;
        n=scn.nextInt();
        for(i=0;i<n;i++)
        {
            for(j=0;j<=i;j++)
            {
                System.out.print("*\t");
            }
             System.out.println();
        }

        // write ur code here

    }
}